enclave-manager/web/src/emui/enclaves/Enclaves.tsx

```diff
@@ -0,0 +1,27 @@
+import { Flex } from "@chakra-ui/react";
+import { Outlet, RouteObject } from "react-router-dom";
+import { Enclave, enclaveLoader, EnclaveLoaderReturnType } from "./Enclave";
+import { EnclaveBreadcrumbs } from "./EnclaveBreadcrumbs";
+import { EnclaveList } from "./EnclaveList";
+
+export const enclaveRoutes: RouteObject[] = [
+  {
+    path: "/",
+    element: (
+      <Flex direction={"column"} gap={"36px"} width={"100%"}>
+        <EnclaveBreadcrumbs />
+        <Outlet />
+      </Flex>
+    ),
+    handle: { name: () => "Enclaves" },
+    children: [
+      { path: "/", element: <EnclaveList /> },
+      {
+        path: "/enclave/:enclaveName",
```

Comment:
When the EM UI runs embedded using the cloud gateway URL, e.g. `cloud.kurtosis.com/gateway/ips/1-2-3-4/ports/9711` the path becomes prepended with all that junk so absolute paths become problematic. Basically the EM UI has to ignore whatever junk is prepended to the paths that it cares about. We have a setting enabled in the EM UI today that does that but it also creates issues. This is an important thing to work through to support the "embedded use case"

Reply:
I think this will still be fine in this file, but point taken - I think it will have to be handled in App.tsx.

It would be great to go through how to test the embedded functionality on Thursday if possible?